pkg/config/control_plane.go

```diff
@@ -314,6 +314,14 @@ type FileStoreGCSConfig struct {
 }
 
 type FileStoreS3Config struct {
+	// The bucket name to store artifacts and logs in the pipe.
```

Comment:
typo: `pipe` -> `piped`

Reply:
thank you, addressed on https://github.com/pipe-cd/pipe/pull/1485/commits/8dea680e82b2a373886f4c69bebc9c9fee066b03 🙏